src/UserAccount.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const UserAccount = ({ userId }) => {
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const response = await axios.get(`/api/user/${userId}`);
+        setUser(response.data);
+      } catch (error) {
+        console.error('Error fetching user data:', error);
+      }
+    };
+
+    fetchUserData();
+  }, [userId]);
+
+  const addBookmark = async (newBookmark) => {
+    try {
+      const response = await axios.put(`/api/user/${userId}`, { bookmark: newBookmark });
+      setUser(response.data);
+    } catch (error) {
+      console.error('Error adding bookmark:', error);
+    }
+  };
+
+  if (!user) return <p>Loading...</p>;
+
+  return (
+    <div>
+      <h1>User Account</h1>
+      <UserProfile user={user} />
+      <BookmarkedEvents events={user.bookmarks} addBookmark={addBookmark} />
+    </div>
+  );
+};
+
+const UserProfile = ({ user }) => (
+  <div>
+    <h2>{user.firstName} {user.lastName}</h2>
+    <p>Email: {user.email}</p>
+    <p>Bio: {user.bio}</p>
+  </div>
+);
+
+const BookmarkedEvents = ({ events, addBookmark }) => {
+  const [newEvent, setNewEvent] = useState({
+    title: '',
+    startDate: '',
+    endDate: '',
+    address: '',
+    coordinates: { lat: '', lng: '' }
+  });
+
+  const handleAddBookmark = () => {
+    addBookmark(newEvent);
+    setNewEvent({
+      title: '',
+      startDate: '',
+      endDate: '',
+      address: '',
+      coordinates: { lat: '', lng: '' }
+    });
+  };
+
+  return (
+    <div>
```

Comment:
Would encourage you to leverage the common components in the `/components` folder for page layout and styling. Feel free to look at other pages (e.g., the contact us page) for examples on how to leverage

Reply:
Specifically look at implementing the `ContentContainer` component to surround the page